### DISCUSSION TITLE
How to sort field in form

### DISCUSSION THREAD

**Discussion by aqilmujahidiin** [filamentphp/filament#15107] [Help]
### Package

Form builder

### Package Version

v3.2

### How can we help you?

![image](https://github.com/user-attachments/assets/81ffa2f8-704d-4275-81e4-180f871b5dbe)

I've done this trick, using grid and fieldset, but it's not satisfactory, I don't want to use sections or fieldsets, just form a schema but the fields are sequential as in the picture, is this possible? if yes, how? thanks

**Comment by leandrocfe** [comment]
use `Group`

```php
use Filament\Forms\Components\Split;
use Filament\Forms\Components\Group;

Split::make([
    Group::make([
        TextInput::make('field1'),
        TextInput::make('field2'),
        TextInput::make('field3'),
    ]),
    Group::make([
        TextInput::make('field4'),
        TextInput::make('field5'),
    ]),
    Group::make([
        TextInput::make('field6'),
        TextInput::make('field7'),
        TextInput::make('field8'),
    ]),
]),
```
